### PR TITLE
Add margin to Next page plugin, and install new version of Sticky banner

### LINKED ISSUE #321: fix css in doc-page and sidebar

### PR DESCRIPTION
**Pull request by JoshK2**
closed #321 
- add margin on next-page-plugin.
- install new version of sticky banner with these changes:
   - change local storage key name to be more specific.
   - change header tag to div element.

### CODE REVIEW

**Comment by ranm8** [comment]
@odedre can u help review and give feedback?

**Comment by ranm8** [comment]
@odedre can u help review and give feedback?

File: docs/blocks/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -28,9 +28,20 @@ export type SidebarProps = {
 
 export function Sidebar({ primaryLinks, sections, className, linkPrefix, ...rest }: SidebarProps) {
   const [isOpen, setIsOpen] = useState(false);
+  const [isStickyBannerOpen, setStickyBanner] = useState(false);
```
Comment (ranm8) [inline docs/blocks/sidebar/sidebar.tsx:31]: why is that here?

File: docs/blocks/sidebar/sidebar.module.scss [hunk 2]
```diff
@@ -13,7 +13,10 @@
     position: fixed;
     overflow-y: hidden;
     box-sizing: border-box;
-    height: calc(100% - 41px);
+    height: 100%;
+  }
+  &.isStickyBannerOpen > div:last-child {
```
Comment (ranm8) [inline docs/blocks/sidebar/sidebar.module.scss:18]: Why we are coupling the sticky banner with the sidebar?

File: docs/blocks/sidebar/sidebar.tsx (hunk 1 above)
Comment (JoshK2) [reply docs/blocks/sidebar/sidebar.tsx:31]: We need this for the css, but I think @KutnerUri solve the issue with the layout and I can remove this from here.
But I'm waiting for @KutnerUri PR to be merged.

File: docs/blocks/sidebar/sidebar.module.scss (hunk 2 above)
Comment (JoshK2) [reply docs/blocks/sidebar/sidebar.module.scss:18]: same, I want to remove this and I'm waiting for @KutnerUri PR to be merged.

**Comment by NitsanCohen770** [comment]
@JoshK2 is this PR still relevant or was it fixed by someone else?
If it's still relevant can you please fix the conflicts?
We are moving bit.dev to lanes.

**Comment by JoshK2** [comment]
> @JoshK2 is this PR still relevant or was it fixed by someone else? If it's still relevant can you please fix the conflicts? We are moving bit.dev to lanes.

I'm working on it now.
